### PR TITLE
Remove riak

### PR DESCRIPTION
**Pull request by pawlooss1**
This PR removes Riak completely from repository.

### CODE REVIEW

**Comment by mongoose-im** [comment]
[small_tests_25_arm64](https://circleci.com/gh/esl/MongooseIM/177250) / small_tests / 46475219fd407b4261a63e238cc0e43a71eb8ce9
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4035/177250/small_tests.25.3-arm64) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/4035/177250/small_tests.25.3-arm64/small/ct_run.mongooseim@localhost.2023-06-12_11.01.11/lib.mongooseim.logs/run.2023-06-12_11.01.12/suite.log.html)

**Comment by mongoose-im** [comment]
[elasticsearch_and_cassandra_25](https://circleci.com/gh/esl/MongooseIM/177267) / elasticsearch_and_cassandra_mnesia / 9706c240c7a0b8d11770ee8cfbcd5103ef159edd
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4035/177267/elasticsearch_and_cassandra_mnesia.25.3-amd64)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/177286) / small_tests / 04662e71b68cec0910ba4861515a17385354274f
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4035/177286/small_tests.24.3.4.9-amd64) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/4035/177286/small_tests.24.3.4.9-amd64/small/ct_run.mongooseim@localhost.2023-06-12_11.18.39/lib.mongooseim.logs/run.2023-06-12_11.18.40/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/177606) / small_tests / 364d0be552c8dcce51ad99060a65b6349c7060c5
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4035/177606/small_tests.24.3.4.9-amd64) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/4035/177606/small_tests.24.3.4.9-amd64/small/ct_run.mongooseim@localhost.2023-06-14_10.12.00/lib.mongooseim.logs/run.2023-06-14_10.12.00/suite.log.html)

File: tools/mam_search_schema.xml [hunk 1]
```diff
@@ -43,15 +43,6 @@
         <!-- If there is a sibling, use vtag to differentiate them -->
         <field name="_yz_vtag" type="_yz_str" indexed="true" stored="false" multiValued="false"/>
 
-        <!-- Riak Key: The key of the Riak object this doc corresponds to. -->
```
Comment (chrzaszcz) [inline tools/mam_search_schema.xml:46]: I think this whole file can be removed.

**Review by chrzaszcz — COMMENTED** [review]
Looks good, I only have two minor comments 🔥

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/177989) / small_tests / 96d8a830bbcf02996d6e6e705a66ef4308044ea8
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/4035/177989/small_tests.24.3.4.9-amd64) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/4035/177989/small_tests.24.3.4.9-amd64/small/ct_run.mongooseim@localhost.2023-06-15_11.03.55/lib.mongooseim.logs/run.2023-06-15_11.03.56/suite.log.html)

**Pull request merged**